test/unittests/CMakeLists.txt

```diff
@@ -195,6 +292,7 @@ endif (BUILD_UNITTESTS)
 if (BUILD_UNITTESTS_DEBUG)
   add_executable (${PROJECT_TEST_DEBUG_NAME} ${CVMFS_UNITTEST_DEBUG_SOURCES})
   add_dependencies (${PROJECT_TEST_DEBUG_NAME} googletest)
+  add_dependencies (${PROJECT_TEST_DEBUG_NAME} libvjson)
```

Comment:
This is a necessary dependency to compile json_document.cc, which is anyway at the moment not compiled.